docs/synchronizations-faq.md

```diff
@@ -0,0 +1,27 @@
+# Synchronizations FAQ
+
+In Private Packagist synchronizations is how you keep packages, teams, their members, and access permissions in sync with your version control system like GitHub, GitLab or Bitbucket.
+
+You can find below answers for frequently-asked questions concerning Synchronizations.
+
+## How to migrate my synchronization from GitHub to Bitbucket?
```

Comment:
Need to check this on the page but I think the first h2 from the md file gets removed and added as sub header which would break this here.